commands/serverless-extra.go

```diff
@@ -151,7 +155,17 @@ func RunServerlessExtraGetMetadata(c *CmdConfig) error {
 	if err != nil {
 		return err
 	}
-	output, err := RunServerlessExec(projectGetMetadata, c, []string{flagJSON}, []string{flagEnv, flagInclude, flagExclude})
+	r, _ := c.Doit.GetBool(c.NS, flagProjectReader)
+
+	var output do.ServerlessOutput
+	project := do.ServerlessProject{
+		ProjectPath: args[0],
```

Comment:
Sorry.   My previous suggestion was slightly broken.   That's the problem with the github suggestion thing.  It encourages the  reviewer to propose (and the PR author to accept) untested code.

```suggestion
		ProjectPath: c.Args[0],
```

Reply:
Not a problem, I noticed that and fixed and tested it in the local system.